scripts/update_project_mailing_lists.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+""" Cron script that attempts to ensure synchronicity between
+    remote mailing lists on MailGun and their local representations.
+"""
+import logging
+
+from modularodm import Q
+
+from framework.celery_tasks import app as celery_app
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.mailing_list.utils import full_update
+from website.project.model import Node
+
+logger = logging.getLogger(__name__)
+
+def main(dry_run=True):
+    if dry_run:
+        def full_update(*args, **kwargs):
```

Comment:
Does this work?
`full_update` will only be defined as an empty function in the scope of `main`.
Anywhere else it *should* be the original one from `website.mailing_list.utils`